src/components/common/Map.tsx

```
@@ -0,0 +1,23 @@
+import { useState } from "react";
+
+import MapImage from "@/assets/images/map.png";
+
+import MapModal from "@/components/map/Modal";
+
+const Map = ({ isNumber }: { isNumber: number }) => {
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)